qa/rpc-tests/getrawtransaction_insight.py

```diff
@@ -27,8 +27,13 @@ def __init__(self):
     def setup_network(self):
         # -insightexplorer causes spentindex to be enabled (fSpentIndex = true)
 
-        self.nodes = start_nodes(self.num_nodes, self.options.tmpdir,
-            [['-debug', '-txindex', '-experimentalfeatures', '-insightexplorer']] * self.num_nodes)
+        self.nodes = start_nodes(self.num_nodes, self.options.tmpdir, extra_args=[[
+            '-debug',
+            '-txindex',
+            '-experimentalfeatures',
```

Comment:
Here we pass `-experimentalfeatures` explicitly, which is different to other calls to `start_nodes`.

Reply:
`-experimentalfeatures` here is for `-insightexplorer`, and was pre-existing.